lib/base32.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+module.exports = (thirtyTwo) => {
+    /**
+     * Decode data from Base32.
+     *
+     * @param {(Buffer|string)} data
+     * @return {Buffer} decoded
+     */
+    function decode(data) {
+        return thirtyTwo.decode(data);
+    }
+
+
+    /**
+     * Decodes data that was encoded as Base32 for URIs.
+     *
+     * @param {string} data
+     * @return {Buffer} decoded
+     */
+    function decodeForUri(data) {
+        var copy;
+
+        copy = data;
+
+        if (copy.length % 8) {
+            copy += "=======".substr(copy.length % 8 - 1);
+        }
+
+        return decode(copy);
+    }
+
+
+    /**
+     * Encode data into Base32.
+     *
+     * @param {(Buffer|string)} data
+     * @return {string} encoded
```

Comment:
At first glance this would look to return the variable `encoded`, but this returns the encoded string.